content/docs/start/data-and-model-versioning.md

```diff
@@ -223,43 +221,40 @@ $ dvc checkout
 
 <details>
 
-### ⚙️ Expand to get the previous version of the dataset.
+### ⚙️ Expand to get the previous version of the dataset 🕑
 
-Let's cleanup the previous artificial changes we made and get the previous :
+Let's go back to the original version of the data:
 
 ```dvc
-$ git checkout HEAD^1 data/data.xml.dvc
+$ git checkout HEAD~1 data/data.xml.dvc
```

Comment:
Is `HEAD~1` more common than `HEAD^`? Either one works, up to you. Just a thought

Reply:
`^` is an escape char in Windows so I'd avoid using it :)

Reply:
Good thinking! Will keep it in mind.

Reply:
:no_entry_sign: :wind_face: :door: :door: (no-wind-doors) :P

Reply:
I can't believe there's no window emoji.